riocli/device/util.py

```diff
@@ -86,7 +86,7 @@ def find_device_guid(client: Client, name: str) -> str:
     raise DeviceNotFound()
 
 def find_device_by_name(client: Client, name: str) -> Device:
-    devices = client.get_all_devices(device_name=name)
+    devices = client.get_all_devices()
```

Comment:
@RomilShah Please don't change this. The device manager API now supports filtering device list by name. This code was updated to save on iterating over the list.

What issue did you face that you had to revert this?